livekit-server/templates/NOTES.txt

```diff
@@ -11,9 +11,12 @@ Please ensure that the following ports on the nodes are open on your firewall.
 {{- end }}
 * WebRTC TCP {{ .Values.livekit.rtc.tcp_port }}
 {{- if .Values.livekit.turn.enabled }}
-{{- if ne "disabled" .Values.livekit.turn.loadBalancerType }}
+{{- if .Values.livekit.turn.loadBalancerType }}
```

Comment:
what if `loadBalancerType` is set to "disabled" ? Don't think we should assume TURN/TLS is turned on?